pkg/services/app/add.go

```diff
@@ -73,6 +74,10 @@ func (a *AppSvc) Add(configGit git.Git, gitProvider gitproviders.GitProvider, pa
 		return err
 	}
 
+	if strings.HasPrefix(params.Name, "wego-") {
```

Comment:
The check doesn't match the error message.  I was able to create an app with just `wego` - it doesn't cause an issue but the check needs to match the error.